starter-code/index.html

```diff
@@ -12,23 +12,35 @@
     <!-- Code Fellows favicon -->
     <link rel="icon" type="image/x-icon" href="https://i.imgur.com/7v5ASc8.png">
     <!--
-  TODO: Complete this script tag below to hold the Handlebars template.
+  TODO: DONE! Complete this script tag below to hold the Handlebars template.
     - Be sure to give it a meaningful id attribute, and don't forget to
       specify the type!
     - Insert correct placeholder properties based on our Model.
     -->
-    <script id=" " type=" ">
-      <article class="clearfix" data-author=" " data-category=" ">
-        <header>
-          <h1> </h1>
-          <div class="byline">
-            By <address><a href=" "> </a></address>
-            {{publishStatus}}
-          </div>
-        </header>
-        <section class="article-body"> </section>
-        <a href="#" class="read-on">Read on &rarr;</a>
-      </article>
+    <script id="articleTemplate" type="text/x-handlebars-template">
+      {{#each this}}
+        <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
+          <header>
+            <h1> {{title}} </h1>
+            <div class="byline">
+              By <address> {{author}} <a href="{{authorUrl}}"> </a></address>
+              {{publishStatus}}
+            </div>
+          </header>
+          <section class="article-body"> {{{body}}} </section>
+          <a href="#" class="read-on">Read on &rarr;</a>
+        </article>
+      {{/each}}
+    </script>
+
+    <script id="authorOptions" type="text/x-handlebars-template">
+      <option value="{{value}}"> {{optionName}} </option>
+    </script>
+
+    <script id="categoryOptions" type="text/x-handlebars-template">
+      <option value="{{value}}"> {{optionName}} </option>
```

Comment:
One suggestion - you'll probably want both the value and the text within the option tag to reflect the author or category, like this:
```
<script id="author-filter-template" type="text/x-handlebars-template">
<option value="{{author}}">{{author}}</option>
</script>

<script id="category-filter-template" type="text/x-handlebars-template">
<option value="{{category}}">{{category}}</option>
 </script>